ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPage.java

```diff
@@ -20,7 +21,9 @@
 	SELECT_START_CONNECTION(new IntegrationConnectionSelectComponentStart()),
 	SELECT_FINISH_CONNECTION(new IntegrationConnectionSelectComponentFinish()),
 	CHOOSE_A_FINISH_CONNECTION(new IntegrationConnectionSelectComponentFinish()),
-	ADD_TO_INTEGRATION(new IntegrationSaveOrAddStepComponent());
+	ADD_TO_INTEGRATION(new IntegrationSaveOrAddStepComponent()),
+	PERIODIC_SQL_INVOCATION(new ActionConfigureComponentPeriodicSql()),
+	INVOKE_SQL(new ActionConfigureComponentPeriodicSql()),;
```

Comment:
There should be probably different class.

Reply:
@lvydra could you be more specific, please.

Reply:
I guess this line `INVOKE_SQL(new ActionConfigureComponentPeriodicSql()),;`
should be `INVOKE_SQL(new ActionConfigureComponentInvokeSql()),;`